docs/source/guide/storage.md

```diff
@@ -311,5 +320,13 @@ You can specify additional parameters from the Label Studio UI.
 | regex | Specify a regular expression to filter directory objects. Use ".*" to collect all objects. | Skips all directory objects. |
 | use_blob_urls | If true, treat every directory object as a source file. Use for resources like JPG, MP3, or similar file types. If false, directory objects are interpreted as tasks in Label Studio JSON format with one object per task. | false |
 
+### Set up local storage with Docker
```

Comment:
why it nearly duplicates what's written in start.md ?

Reply:
good catch, I'll link to the start.md file instead here.